test/e2e/common.go

```diff
@@ -33,15 +33,16 @@ import (
 
 // Test suite constants for e2e config variables.
 const (
-	KubernetesVersionManagement  = "KUBERNETES_VERSION_MANAGEMENT"
-	KubernetesVersion            = "KUBERNETES_VERSION"
-	CNIPath                      = "CNI"
-	CNIResources                 = "CNI_RESOURCES"
-	KubernetesVersionUpgradeFrom = "KUBERNETES_VERSION_UPGRADE_FROM"
-	KubernetesVersionUpgradeTo   = "KUBERNETES_VERSION_UPGRADE_TO"
-	EtcdVersionUpgradeTo         = "ETCD_VERSION_UPGRADE_TO"
-	CoreDNSVersionUpgradeTo      = "COREDNS_VERSION_UPGRADE_TO"
-	IPFamily                     = "IP_FAMILY"
+	KubernetesVersionManagement               = "KUBERNETES_VERSION_MANAGEMENT"
+	KubernetesVersion                         = "KUBERNETES_VERSION"
+	CNIPath                                   = "CNI"
+	CNIResources                              = "CNI_RESOURCES"
+	KubernetesVersionUpgradeFrom              = "KUBERNETES_VERSION_UPGRADE_FROM"
+	KubernetesVersionUpgradeTo                = "KUBERNETES_VERSION_UPGRADE_TO"
+	EtcdVersionUpgradeTo                      = "ETCD_VERSION_UPGRADE_TO"
+	CoreDNSVersionUpgradeTo                   = "COREDNS_VERSION_UPGRADE_TO"
+	IPFamily                                  = "IP_FAMILY"
+	KubernetesVersionUpgradeMachineTemplateTo = "KUBERNETES_VERSION_UPGRADE_MACHINE_TEMPLATE_TO"
```

Comment:
```suggestion
	MachineTemplateUpgradeTo = "MACHINE_TEMPLATE_UPGRADE_TO"
```
nit: I think it reads better in this order / then follow the same pattern as `KUBERNETES_VERSION_UPGRADE_TO`

Can you also move it directly below `KubernetesVersionUpgradeTo`?

Reply:
Ideally I would prefer a common prefix with the other UPGRADE_TO env var, but given that we can't easily change the existing env var I would prefer the one I proposed. But let's see what @fabriziopandini thinks.

(except if we deprecate the old var and introduce a new one...)

Reply:
+1 to MACHINE_TEMPLATE_UPGRADE_TO
(we strive for a common suffix)